harvard-lts/drs-import-management-service
# Gunicorn nginx

**Gunicorn Nginx*
* * *

**GitHub Issue**: [HDC8](https://app.zenhub.com/workspaces/hdc-62163f60fb08c60011fa4795/issues/harvard-lts/hdc/8)

# What does this Pull Request do?
We want to use nginx and gunicorn in our environments.

# How should this be tested?

Rebuild the container and go to https://localhost:10580/health to see the Ok! You know it is working as you can reach the page and also now it is https (you will get yelled at about a self signed cert, this is expected)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
